src/Twilio/Http/Request.cs

```diff
@@ -49,9 +64,9 @@ public class Request
         public Request(HttpMethod method, string url)
         {
             Method = method;
-            _uri = new Uri(url);
-            _queryParams = new List<KeyValuePair<string, string>>();
-            _postParams = new List<KeyValuePair<string, string>>();
+            Uri = new Uri(url);
+            QueryParams = new List<KeyValuePair<string, string>>();
+            PostParams = new List<KeyValuePair<string, string>>();
```

Comment:
Why the rename?

Reply:
Fits more with convention since they are now auto-properties.